scripts/format_translations.py

```diff
@@ -1,19 +1,6 @@
 """ Convert translation spreadsheet (downloaded as a csv file) to json.
     Keys that include a "." indicate nesting, so 'flag.chooseLanguage' is converted to {flag:{chooseLanguage: value}}
 
-    Steps:
```

Comment:
It looks like this script *could* still be run in standalone mode. 

I agree about removing the instructions in favor of the README (always prefer 1 clear authoritative source of information) but the usage example/ sample command might still be useful.